docs/introduction/developing-with-threejs.md

```diff
@@ -213,26 +213,26 @@ accomplished with matrices, but three.js provides helpers to make them easier.
 
 Normally, we'd need to call `.updateMatrixWorld ()` on parent `Object3D`s, but
 three.js defaults `Object3D.matrixAutoUpdate` to `true`. We can use three.js's
-`.getWorldPosition ()` and `.getWorldRotation ()`.
+`.getWorldPosition (vector)` and `.getWorldQuaternion (quaternion)`.
 
 To get the world position of an `Object3D`:
 
 ```js
-entityEl.object3D.getWorldPosition();
```

Comment:
should have `()` and probably better to show if the vector/quat is created in a variable in passed in so people know to cache the objects.